python/huggingface_server.Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN pip3 install vllm==${VLLM_VERSION}
 
 FROM nvidia/cuda:12.4.1-runtime-ubuntu22.04 AS prod
 
-RUN apt-get update -y && apt-get install python3.10-venv build-essential gcc python3-dev -y && apt-get clean && \
+RUN apt-get update -y && apt-get install python3.10-venv build-essential gcc python3-dev curl -y && apt-get clean && \
```

Comment:
Adding curl is to run the health check right?

Reply:
Yes

Reply:
isn't better to use ray python api to check the nodes statuses?

Reply:
it is about model runtime health check. not ray.